conftest.py

```diff
@@ -42,8 +36,13 @@ def get_task_ids(user: str=None):
 
 
 def get_task_logs_for_id(task_id: str, lines: int=1000000):
-    return subprocess.check_output([
-        'dcos', 'task', 'log', task_id, '--lines', str(lines)]).decode()
+    try:
+        task_logs = subprocess.check_output([
+            'dcos', 'task', 'log', task_id, '--lines', str(lines), 'stderr'
+        ]).decode()
+        return task_logs
+    except subprocess.CalledProcessError:
+        return None
```

Comment:
Note that we now fail silently. Would some logging output be useful to see what caused the failure?